scipy/spatial/transform/rotation.py

```diff
@@ -417,5 +524,67 @@ def from_euler(cls, seq, angles, degrees=False):
             raise ValueError("Expected angles to have shape (num_rotations, "
                              "num_axes), got {}.".format(angles.shape))
 
-        quat = _elementary_quat_compose(seq, angles, intrinsic)
+        quat = elementary_quat_compose(seq, angles, intrinsic)
         return cls(quat[0] if is_single else quat, normalized=True)
+
+    def as_euler(self, seq, degrees=False):
+        """Return the Euler angles representation of the Rotation.
+
+        This function returns a numpy.ndarray of shape (N, 3) or (3,) depending
+        on how the object was initialized. The algorithm presented in [2]_ has
+        been adapted for our use to extract Euler angles. The paper presents an
+        algorithm for extracting Euler angles from transformation matrices, as
+        opposed to rotation matrices. Thus, the matrix representation used in
+        the paper is a transpose of the direction cosine matrix representation
+        returned by the `as_dcm` function.
+
+        The returned angles are in the range:
+
+            - First angle belongs to [-180, 180] degrees (both inclusive)
+            - Third angle belongs to [-180, 180] degrees (both inclusive)
+            - Second angle belongs to:
+
+                - [-90, 90] degrees if all axes are unique (like xyz)
+                - [0, 180] degrees if all axes are not unique (like zxz)
```

Comment:
I would write "if the thirst and the third axes are the same".

Reply:
![image](https://user-images.githubusercontent.com/15844796/40881904-4d9ef2da-66f0-11e8-876c-d44f6774612b.png)

Yup, it renders fine on my setup.